build/goreleaser/linux/centos_8_arm64.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-arm64
       - linux-service-arm64
     package_name: newrelic-infra
-    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el8.{{ .Arch }}"
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el8.{{ .Arch }}{{ .Env.FIPS }}"
```

Comment:
Same, this should be :

`"newrelic-infra{{ .Env.FIPS }}-{{ .Env.TAG }}-1.el8.{{ .Arch }}"`